packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
This is a client-affecting change, and this needs a version bump

Reply:
ok , 5.2.2